### PR TITLE
Update .gitattributes

### PR DESCRIPTION
**Pull request by erikn69**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks for the PR. Can you put this in the file, alphabetically, please?

File: .gitattributes [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 /.gitignore export-ignore
 /.travis.yml export-ignore
+/Makefile export-ignore
 /phpstan-baseline.neon export-ignore
 /phpstan.src.neon.dist export-ignore
 /phpstan.tests.neon.dist export-ignore
 /phpunit.xml.dist export-ignore
 /psalm.xml export-ignore
 /README.md export-ignore
+/tests export-ignore
```
Comment (GrahamCampbell) [inline .gitattributes:14]: Please leave tests alone. I did directories first, alphabetically, then files, alphabetically.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (erikn69) [reply .gitattributes]: Done

**Comment by GrahamCampbell** [comment]
Thanks.

**Pull request merged**